mantrachain/versions.json

```diff
@@ -3,18 +3,18 @@
   "chain_name": "mantrachain",
   "versions": [
     {
-      "name": "1",
-      "recommended_version": "1.0.0-rc3",
+      "name": "v1",
+      "recommended_version": "1.0.3",
       "compatible_versions": [
-        "1.0.0-rc3"
+        "1.0.3"
```

Comment:
Is "1.0.0-rc3" compatible with "1.0.3"? It is best to build a history of versions rather than re-write the current version.